### PR TITLE
Changes to solve lint issues in CloudConnectionStatus.

### PR DESCRIPTION
**Pull request by MMaiero**
Signed-off-by: Maiero <matteo.maiero@eurotech.com>

### CODE REVIEW

File: kura/org.eclipse.kura.core.status/src/main/java/org/eclipse/kura/core/status/runnables/BlinkStatusRunnable.java [hunk 1]
```diff
@@ -11,29 +11,36 @@
...
-        while (true) {
+        while (this.enabled) {
             try {
                 this.ledManager.writeLed(true);
                 Thread.sleep(CloudConnectionStatusEnum.SLOW_BLINKING_ON_TIME);
                 this.ledManager.writeLed(false);
                 Thread.sleep(CloudConnectionStatusEnum.SLOW_BLINKING_OFF_TIME);
-            } catch (InterruptedException | KuraException ex) {
+            } catch (Exception ex) {
                 break;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.core.status/src/main/java/org/eclipse/kura/core/status/runnables/BlinkStatusRunnable.java]: Is it possible to catch a more specific exception?

Comment (MMaiero) [reply kura/org.eclipse.kura.core.status/src/main/java/org/eclipse/kura/core/status/runnables/BlinkStatusRunnable.java]: Change made

**Comment by pierantoniomerlino** [comment]
@MMaiero Could you do the same modification also on HeartbeatStatusRunnable and OnOffStatusRunnable?

**Comment by MMaiero** [comment]
Done

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**